bundles/org.openhab.binding.shelly/README.md

```diff
@@ -514,9 +536,31 @@ The Thing id is derived from the service name, so that's the reason why the Thin
 |          |button       |Trigger  |yes      |Event trigger, see section Button Events                                         |
 |meter     |currentWatts |Number   |yes      |Current power consumption in Watts                                               |
 |          |lastPower1   |Number   |yes      |Energy consumption for a round minute, 1 minute  ago                             |
-|          |totalKWH     |Number   |yes      |Total energy consumption in Watts since the device powered up (resets on restart)|
+|          |totalKWH     |Number   |yes      |Total energy consumption in kw/h since the device powered up (resets on restart)|
 |          |lastUpdate   |DateTime |yes      |Timestamp of the last measurement                                                |
 
+### Shelly 2 - roller mode thing-type: shelly2-roller)
+
+|Group     |Channel      |Type     |read-only|Description                                                                           |
+|----------|-------------|---------|---------|--------------------------------------------------------------------------------------|
+|roller    |control      |Rollershutter|r/w  |can be open (0%), stop, or close (100%); could also handle ON (open) and OFF (close)  |
+|          |input        |Switch   |yes      |ON: Input/Button is powered, see General Notes on Channels                            |
+|          |event        |Trigger  |yes      |Roller event/trigger with payload ROLLER_OPEN / ROLLER_CLOSE / ROLLER_STOP            |
+|          |rollerpos    |Number   |r/w      |Roller position: 100%=open...0%=closed; gets updated when the roller stops, see Notes |
+|          |rollerFav    |Number   |r/w      |Select roller position favorite (1-4, 0=no), see Notes                                |
+|          |state        |String   |yes      |Roller state: open/close/stop                                                         |
+|          |stopReason   |String   |yes      |Last stop reasons: normal, safety_switch or obstacle                                  |
+|          |safety       |Switch   |yes      |Indicates status of the Safety Switch, ON=problem detected, powered off               |
+|meter     |currentWatts |Number   |yes      |Current power consumption in Watts                                                    |
+|          |lastPower1   |Number   |yes      |Accumulated energy consumption in Watts for the full last minute                      |
+|          |totalKWH     |Number   |yes      |Total energy consumption in kw/h since the device powered up (reset on restart)      |
+|          |lastUpdate   |DateTime |yes      |Timestamp of the last measurement                                                     |
+
+`Note: The Roller should be calibrated using the device Web UI or Shelly App, otherwise the position can't be set.`
```

Comment:
To put emphasis on this note you can sourround it with "*" instead of using fixed-width font, which is more for code.

Reply:
fixed